### ISSUE TITLE
Prisma Cloud Vulnerability report

### ISSUE THREAD

**Issue by medical-devices** [kubernetes-sigs/aws-load-balancer-controller#3252]
**Describe the bug**

I am getting a report of a security violation:

```
Prisma Cloud Vulnerability report for image public.ecr.aws/eks/aws-load-balancer-controller:v2.5.1 (triggered by Image Vulnerabilities - CORE)
```

detailed at https://nvd.nist.gov/vuln/detail/CVE-2023-29402

I am using v2.5.1. From the release notes it does not look like this was fixed in v2.5.2.

Is that correct?

**Steps to reproduce**

Scan binary with prisma cloud

**Expected outcome**

No vulnerability reported

**Environment**

* AWS Load Balancer controller version: v2.5.1
* Kubernetes version: 1.23
* Using EKS (yes/no), if so version? yes, 1.23

**Additional Context**:

**Comment by oliviassss** [comment]
@medical-devices thanks for the report, we will address this CVE in the next patch release.

**Comment by oliviassss** [comment]
@medical-devices, we have addressed the CVEs in the release v2.5.3, I'm closing this issue for now. Feel free to reach out or reopen if you have any issue. Thanks.
